Scripts/configure-ODBC-DSN.ps1

```diff
@@ -5,31 +5,52 @@
 
 $DSNNames = (Get-OdbcDsn).Name
 
-# AZURESQL
 
-Write-Host("Updating Server name and database name for AZURESQL")
+# AZURESQL
 
 if($DSNNames -contains "AZURESQL" -and (Get-OdbcDsn -Name "AZURESQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for AZURESQL")
+
     $sqlserver = "db-regression-" + $lansaVersion + ".database.windows.net" 
     Set-OdbcDsn -Name "AZURESQL" -DsnType "System" -Platform "32-bit" -SetPropertyValue @("Server=$sqlserver", "Database=$lansaVersion")
+
 }
 
+
 # Oracle
 
-Write-Host("Updating Server name for Oracle")
+if($DSNNames -contains "ora19cdb" -and (Get-OdbcDsn -Name "ora19cdb").Platform -eq "32-bit"){
+
+    $TNSNamesPath = "C:\oracle\instantclient_21_6\network\admin\tnsnames.ora"
+    $Content = Get-Content $TNSNamesPath
+    $oraServer = "ora" + $lansaVersion + ".cnyed5gpqwey.us-east-1.rds.amazonaws.com"
+
+    Write-Host("Updating HOST name for Oracle in $TNSNamesPath to $oraServer")
 
-if($DSNNames -contains "ORACLE" -and (Get-OdbcDsn -Name "ORACLE").Platform -eq "32-bit"){
-    $oraServer = "ora" + $lansaVersion
-    Set-OdbcDsn -Name "ORACLE" -DsnType "System" -Platform "32-bit" -SetPropertyValue "ServerName=$oraServer"
+
+    foreach ($string in (Get-Content $TNSNamesPath))
+    {
+        if($string.Contains("(ADDRESS=(PROTOCOL=TCP)(HOST=")){
+
+            $newString = "            (ADDRESS=(PROTOCOL=TCP)(HOST=ora" + $lansaVersion + ".cnyed5gpqwey.us-east-1.rds.amazonaws.com)(PORT=1521))"
+            $Content = $Content.replace($string,$newString)
+            
+        }
+    }
+    $Content | Set-Content -Path $TNSNamesPath
 }
 
-# MYSQL
 
-Write-Host("Updating Server name and database name for MYSQL")
+# MYSQL
 
 if($DSNNames -contains "MYSQL" -and (Get-OdbcDsn -Name "MYSQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for MYSQL")
```

Comment:
You MUST output the values you are updating the ODBC DSN with.

Reply:
Done.